identity/verifier.go

```diff
@@ -2,26 +2,52 @@ package identity
 
 import (
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/syndtr/goleveldb/leveldb/errors"
 )
 
 type Verifier interface {
 	Verify(message []byte, signature Signature) bool
 }
 
-type ethereumVerifier struct {
-	peerIdentity Identity
+func NewVerifyIsAuthorized() *verifyIsAuthorized {
```

Comment:
"verify" english verb, verifyIsAuthorized - verify action is allowed ? This name is very confusing

Reply:
Renamed to `NewVerifierAuthorization` and `NewVerifierIdentity`